scripts/import_osm.py

```diff
@@ -0,0 +1,111 @@
+import argparse
+import sys
+import osmnx as ox
+from ipygis import get_connection_url
+from sqlalchemy import create_engine
+from sqlalchemy.engine.base import Engine
+from sqlalchemy.schema import CreateSchema
+from geoalchemy2 import Geometry, WKTElement
+from typing import Dict
+
+# test simple import now, convert to module later
+sys.path.insert(0, "..")
+from models import OSMPoint
+from osm_tags import tags_to_filter
+
+
+class OsmImporter(object):
+    """Import OSM data to PostGIS using Overpass API."""
+
+    def __init__(self, args: Dict):
+        args = self._parse_args(args)
+        self.slug = args["slug"]
+        self.bbox = args["bbox"]
+        self.city = args["city_name"]
+        self._engine = self._create_engine()
+
+    @staticmethod
+    def _parse_args(args) -> Dict:
+        """Parse input arguments to a dict."""
+        slug = args["city"]
+        if not slug:
+            raise AssertionError("You must specify the city name.")
+        # BBOX (minx, miny, maxx, maxy) takes precedence over city
+        arg_bbox = args["bbox"]
+        if arg_bbox:
+            minx, miny, maxx, maxy = map(float, arg_bbox.split(", "))
+            city_name = ""
+        else:
```

Comment:
Perhaps we don't need these anymore. Probably nobody uses these slugs anyway, so better get rid of useless stuff, they are old leftovers in the accessibility importer too.